lib/pages/chat_list/chat_list_view/chat_list_view_style.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/material.dart';
+
+class ChatListViewStyle {
+  static BorderRadius containerBorderRadius = BorderRadius.circular(16);

```

Comment:
You can use like this to take advantage of const
  static const BorderRadius containerBorderRadius = BorderRadius.all(Radius.circular(16));